SalesforceSDK/NoteSync/NoteSync.Windows/NoteSync.Windows.csproj

```diff
@@ -136,21 +136,21 @@
     </Reference>
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\Salesforce.SDK.App\Salesforce.SDK.App.csproj">
+    <ProjectReference Include="..\..\Salesforce.SDK.App\App.csproj">
```

Comment:
nit: not a big fan of relative path. I would prefer to see these paths based on $(SolutionDir) but would not consider this a blocker. You can take a look at S1 win app code to see how we use $(SolutionDir)